### PR TITLE
Nanopolish implement

### PR DESCRIPTION
**Pull request by replikation**
* implements nanopolish #87  as an option (default is still medaka)
* super slow (7x medaka speed) - not sure if anyone wants to use it
* added #89 suggestion but not as a warning more of a general info
   * its a "info print" showing the latest release found on the repo

### CODE REVIEW

**Comment by hoelzer** [comment]
I started a test run w/ nanopolish:

```
nextflow run poreCov.nf -profile local,singularity -params-file $DIR/run.yml -w $DIR/work --rki -resume --nanopolish --output results_nanopolish
```

```
WARN: Input tuple does not match input set cardinality declared by process `artic_ncov_np_wf:artic_nanopolish` -- offending value: [IMSSC2-4-2021-03085, /mnt/scratch_slow04/sc2/genome-reconstruction/nanopore/2021-03-25/run42/work/81/1bb3c3aa7696e7f8caa84e0716dd50/IMSSC2-4-2021-03085_filtered.fastq.gz, /home/hoelzerm/git/poreCov/data/external_primer_schemes, /mnt/scratch_slow04/sc2/genome-reconstruction/nanopore/2021-03-25/run42/fast5_pass, /mnt/scratch_slow04/sc2/genome-reconstruction/nanopore/2021-03-25/run42/work/c4/356609697c7ed2cee5422b353383f5/fastq_tmp/barcoding_summary.txt, /mnt/scratch_slow04/sc2/genome-reconstruction/nanopore/2021-03-25/run42/work/c4/356609697c7ed2cee5422b353383f5/fastq_tmp/sequencing_summary.txt]
```

and when I look into one of the nanopolish work dirs:

```
(nextflow) hoelzerm@sebio06:/mnt/scratch_slow04/sc2/genome-reconstruction/nanopore/2021-03-25/run42$ ls work/93/efe194196a5f6467249db419b3dfa3/
barcoding_summary.txt  external_primer_schemes  fast5_pass  IMSSC2-4-2021-03217_filtered.fastq.gz  IMSSC2-4-2021-03217.minion.log.txt
(nextflow) hoelzerm@sebio06:/mnt/scratch_slow04/sc2/genome-reconstruction/nanopore/2021-03-25/run42$ cat work/93/efe194196a5f6467249db419b3dfa3/.command.log
error: could not read file sequencing_summary.txt
Running: nanopolish index -s sequencing_summary.txt -d fast5_pass IMSSC2-4-2021-03217_filtered.fastq.gz
Command failed:nanopolish index -s sequencing_summary.txt -d fast5_pass IMSSC2-4-2021-03217_filtered.fastq.gz
(nextflow) hoelzerm@sebio06:/mnt/scratch_slow04/sc2/genome-reconstruction/nanopore/2021-03-25/run42$
```

So the sequncing_summary.txt is missing? 

I only have:

```
(nextflow) hoelzerm@sebio06:/mnt/scratch_slow04/sc2/genome-reconstruction/nanopore/2021-03-25/run42$ ls work/93/efe194196a5f6467249db419b3dfa3/
barcoding_summary.txt  external_primer_schemes  fast5_pass  IMSSC2-4-2021-03217_filtered.fastq.gz  IMSSC2-4-2021-03217.minion.log.txt
```

File: workflows/process/artic.nf
Comment (hoelzer) [inline workflows/process/artic.nf:38]: I think this needs to be `barcoding_summary.txt` in my case?

Comment (replikation) [reply workflows/process/artic.nf:38]: mhhh unsure ill test tomorrow with fast5 with barcodes. but thanks ill test this before merge.

Comment (replikation) [reply workflows/process/artic.nf:38]: @hoelzer can you also show me whats "inside" the workdir?

**Comment by hoelzer** [comment]
I think I copied also an 'ls' command output? I have a soft-linked
"barcode-summary.txt"file

Christian Brandt ***@***.***> schrieb am Do., 25. März 2021,
23:05:

**Comment by replikation** [comment]
* @hoelzer should work now, the bug was that the [barcode summary, sequence summary ] tuple was "de-tupled" and thus the sequence summary file that I need was "lost" / replaced by the barcode summary file

**Review by hoelzer — APPROVED** [review]
works on my end, looks good! 

btw I got for my test run basically the same mutations called for all samples except for one B.1.1.7 where `medaka` additionally (correctly? wrongly?) found ORF1a:Y3607X :) Not sure which one is correct here ;)

**Comment by replikation** [comment]
+ okay thanks, ill merge this and do a release after some more testings

**Pull request merged**
